macosx/PiecesView.mm

```diff
@@ -39,16 +40,29 @@ @interface PiecesView ()
 
 @implementation PiecesView
 
+- (void)drawRect:(NSRect)dirtyRect
+{
+    [[NSColor.controlTextColor colorWithAlphaComponent:0.2] setFill];
+    NSRectFill(dirtyRect);
+    [super drawRect:dirtyRect];
+}
+
 - (void)awakeFromNib
 {
     //store box colors
-    self.fGreenAvailabilityColor = [NSColor colorWithCalibratedRed:0.0 green:1.0 blue:0.4 alpha:1.0];
-    self.fBluePieceColor = [NSColor colorWithCalibratedRed:0.0 green:0.4 blue:0.8 alpha:1.0];
+    self.fGreenAvailabilityColor = NSColor.systemGreenColor;
```

Comment:
OK this is minor, but if we replace those with system colors, we should get rid of the variables `fGreenAvailabilityColor` and `fBluePieceColor`.